apps/helpthechickens.com/src/app/page.tsx

```diff
@@ -61,21 +63,247 @@ const Hero = () => {
   );
 };
 
-const PetitionSection = () => {
+function PetitionSection() {
   return (
     <Section
       className="gap-12 items-center bg-slate-200 xl:rounded-lg py-12 md:px-16"
       id="petition-section"
     >
       <h2 className="font-semibold text-xl uppercase self-start text-center md:text-left w-full">
-        Ask Petaluma City Council to shut down Perdue's slaughterhouse
+        Ask Petaluma City Council to shut down Perdue&apos;s slaughterhouse
       </h2>
       <EmailPetition defaultMessage={DEFAULT_MESSAGE} onSubmit={onSubmit} />
     </Section>
   );
 };
 
-const Footer = () => {
+function MoreBackgroundSection() {
+  function ExposingPerdueYouTubeVideo() {
+    return (
+      <iframe
+        width="560" height="315"
+        src="https://www.youtube.com/embed/3r4xjelwY0U?si=ksKf8aU7aArUnWkr"
+        title="YouTube video player"
+        frameBorder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+        referrerPolicy="strict-origin-when-cross-origin"
+        allowFullScreen>
+      </iframe>
+    );
+  }
+
+  return (
+    <Section>
+      <div className="flex flex-col lg:flex-row items-center justify-evenly gap-12 text-center">
+        <div>
```

Comment:
nit: this seems like a good use for a `p` tag, but i don't really care that much

Reply:
fixed!